relayer/relayer.go

```diff
@@ -26,8 +27,14 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	maxSubscribeAttempts   = 10
+	maxResubscribeAttempts = 1000_000
```

Comment:
It'd require a slight rework in `evm/subscriber.go`, but it'd be cleaner to be able to specify in the `Subscribe` argument list that we want to continue attempting to resubscribe forever, rather than passing in a very large number.